test/transforms.jl

```diff
@@ -598,6 +598,45 @@
     @test isequalmissing(n.e, [missing, 5, 6, 5])
     @test isequalmissing(n.f, [4, missing, 4, 5])
 
+    # column eltype
+    ttypes = Tables.schema(t).types
```

Comment:
move this to after the apply call and remove the redundant comment

Reply:
Made in [1514725](https://github.com/JuliaML/TableTransforms.jl/pull/35/commits/151472520bcb0c3d5222738bfa33dce003172cf7)